storybook/pages/StatusTrayIconPage.qml

```diff
@@ -0,0 +1,39 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import mainui 1.0
+
+Item {
+    id: root
+
+    ColumnLayout {
+        anchors.centerIn: parent
+
+        Label {
+            text: "Current tray icon: " + trayIcon.icon.source
+        }
+        TextField {
+            id: iconField
+            placeholderText: "Enter icon source"
+            onAccepted: trayIcon.icon.source = iconField.text
+        }
+        CheckBox {
+            id: mask
+            checked: trayIcon.icon.mask
+            text: "Using icon mask"
+            onToggled: trayIcon.icon.mask = !trayIcon.icon.mask
+        }
+        CheckBox {
+            id: visible
+            checked: trayIcon.visible
+            text: "Visible"
+            onToggled: trayIcon.visible = !trayIcon.visible
+        }
+    }
+
+    StatusTrayIcon {
+        id: trayIcon
+        onActivateApp: print ("onActivateApp")
```

Comment:
Maybe it would make sense to add a log output component like in AboutViewPage.qml, so "onActivateApp'  is printed there?